.gitattributes

```diff
@@ -2,3 +2,6 @@ graphics/** filter=lfs diff=lfs merge=lfs -text
 docs/design-ui/**/*.png filter=lfs diff=lfs merge=lfs -text
 docs/design-ui/**/*.jpg filter=lfs diff=lfs merge=lfs -text
 docs/design-ui/**/*.jpeg filter=lfs diff=lfs merge=lfs -text
+*.jpg filter=lfs diff=lfs merge=lfs -text
+*.png filter=lfs diff=lfs merge=lfs -text
+*.jpeg filter=lfs diff=lfs merge=lfs -text
```

Comment:
I think these are ideally constrained to the public_site directory I don't want to break workspace logo or something like that